src/Hintman/Suggest/TrailingNewline.hs

```diff
@@ -0,0 +1,19 @@
+module Hintman.Suggest.TrailingNewline
+       ( suggestNoTrailingNewline
+       ) where
+
+import Hintman (LineNumber)
```

Comment:
It's actually should be vice versa. The module with name `Foo.Bar` should never reexport something from `Foo`. Instead you can create `Hintman.Suggest.Core` and put all data types there.